JavaScript/JavaScript.sublime-syntax

```diff
@@ -1363,8 +1370,24 @@ contexts:
     - match: (?={{class_element_name}})
       push: class-element
 
+  static-block:
+    - match: static{{identifier_break}}
```

Comment:
How about `- meta_include_prototype: false` here (and in the next context)?

Reply:
Here, I think it wouldn't do anything. In the next context, adding `meta_include_prototype` could break things:

```js
class C {
  static /**/ {}
}
```

Reply:
Only intented to propose to add it before 1374 as this context is pushed by lookaheads. So any injected/prepended rule would be nonsense.

Reply:
While suggesting this, I have use cases in mind, which inherit JavaScript to inject variable interpolation via `prototype` such as https://github.com/sublimehq/Packages/pull/2797/files#diff-450eddd7ff448ef509c1ae063ca8f11b631dfe871ccc2ecd6bf6656416159260

That's probably a general thing to take into account, which goes beyound the scope of this PR.

Reply:
I think that if we needed a change here, we'd probably need it in other places, and I'd rather burn that bridge when we come to it.